### DISCUSSION TITLE
Why is the xbox crontoller not supported on the move hub?

### DISCUSSION THREAD

**Discussion by minystreem56** [pybricks/support#1761] [General]
I have some ble controllers that I want to controller the move hub with but it is not supported, is there a hardware reason for this or is it just not implmented? Thanks!

**Comment by dlech** [comment]
The Move hub has a different Bluetooth chip from other hubs. We're not sure if it has all of the required functionality needed. Also, the Move hub has significantly less flash memory than other hubs, so there are lots of other features that we have to leave out of the Move hub just to save space so that the firmware will fit on the hub.

**Reply by minystreem56** [reply]
Ok thank you so much! I was just seeing if I should go out and buy a PU controller or not, thank you!
